addons/ace_tweaks/CfgVehicles.hpp

```diff
@@ -0,0 +1,38 @@
+
+class CBA_Extended_EventHandlers;
+
+class CfgVehicles {
+
+    // Medical litter classes
+    class Thing;
+    class ACE_MedicalLitterBase;
+    class ACE_MedicalLitter_painkillers: ACE_MedicalLitterBase {
+        model = "\A3\Structures_F_EPA\Items\Medical\PainKillers_F.p3d";
+    };
+
+    // Treatment items
+    class Item_Base_F;
+    class ACE_painkillersItem: Item_Base_F {
+        scope = 2;
+        scopeCurator = 2;
+        displayName = Painkillers;
+        author = "Wilhelm Haas";
```

Comment:
Who is this?

Reply:
This is a pseudonym I used in a previous unit.